index.js

```diff
@@ -461,4 +460,10 @@ async function checkFileExists(extractTo, checkFileName) {
   return files.includes(checkFileName);
 }
 
-getDependenciesAndCheckCompatibility();
+/**
+ * @description
+ * it is possible to utilize top-level `await` since ES13/NodeJs-14.8.0
+ */
+(async function (){
```

Comment:
Let's leave it at top level await

Reply:
This make sense since nodejs 14 is being supported from Angular 12

Reply:
Unfortunately, we cannot leave it at the top-level as this is only available for es modules, not common modules.

Reply:
It's been working fine so should be ok

Reply:
There was no `await` before :)